pkgs/applications/networking/browsers/librewolf/src.json

```diff
@@ -1,11 +1,11 @@
 {
-  "packageVersion": "134.0.1-1",
+  "packageVersion": "135.0-1",
   "source": {
-    "rev": "134.0.1-1",
+    "rev": "135.0-1",
     "hash": "sha256-YtZb+VaJu+zofIjJUUx6FjVsEb1sCgFsGlxgP++Yki0="
```

Comment:
This PR changes the hash as well: https://github.com/NixOS/nixpkgs/pull/381570

Reply:
Hum yeah it looks like I did not save it. Closing in favor of the other PR.